app/src/main/java/org/oppia/app/topic/TopicFragment.kt

```diff
@@ -11,14 +11,19 @@ import javax.inject.Inject
 
 /** Fragment that contains tabs for Topic. */
 class TopicFragment : InjectableFragment() {
-  @Inject lateinit var topicFragmentPresenter: TopicFragmentPresenter
+  @Inject
```

Comment:
Ditto here and elsewhere.

Reply:
Ktlint is a bit different, it says to be on the next line. Also, Ben discuss the same in yesterday's meeting

Reply:
Got it.